src/runtime_src/core/common/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ add_subdirectory(xdp)
 if(CMAKE_VERSION VERSION_LESS "3.18.0")
   message(WARNING "CMake version is less than 3.18.0, build of submodule aiebu disabled")
 elseif (${XRT_NATIVE_BUILD} STREQUAL "yes")
-  add_subdirectory(aiebu)
+#  add_subdirectory(aiebu)
```

Comment:
Without commenting this line, we will see error like below:

-- building XRT core libs
-- CMAKE_SYSTEM=Windows-10.0.27723
-- CMAKE_INSTALL_PREFIX=C:/develop/iree-amd-aie-dezhi/iree-build/compiler/plugins/iree-aie-xrt-coreutil/aiebu/xilinx
-- Using aie-rt from C:/develop/iree-amd-aie-dezhi/iree-build/compiler/plugins/iree-aie-xrt-coreutil/aiebu
-- Using ELFIO from C:/develop/iree-amd-aie-dezhi/third_party/XRT/src/runtime_src/core/common/aiebu/src/cpp/ELFIO
-- Found Python3: C:/Python313/python.exe (found version "3.13.0") found components: Interpreter
-- Python version: 3.13.0
-- Could NOT find Boost (missing: Boost_INCLUDE_DIR)
-- Boost version:
CMake Error at C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Boost (missing: Boost_INCLUDE_DIR program_options)
Call Stack (most recent call first):
  C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.27/Modules/FindBoost.cmake:2378 (find_package_handle_standard_args)
  C:/develop/iree-amd-aie-dezhi/third_party/XRT/src/runtime_src/core/common/aiebu/src/cpp/aiebu/CMakeLists.txt:7 (find_package)